app/Repositories/BaseRepository.php

```diff
@@ -0,0 +1,326 @@
+<?php
+namespace App\Repositories;
+
+use Illuminate\Support\Facades\DB;
+use App\Repositories\RepositoryInterface;
+
+abstract class BaseRepository implements RepositoryInterface
+{
+
+    /**
+     * @var $model
+     */
+    protected $model;
+    /**
+     * EloquentRepository constructor.
+     */
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.CloseBracketLine</br>**Severity**: error</br>**File**: app/Repositories/BaseRepository.php</br>**Line**: 220</br>The closing parenthesis of a multi-line function declaration must be on a new line